sdk/mixedreality/azure-mixedreality-authentication/azure/mixedreality/authentication/_utils.py

```diff
@@ -42,7 +42,7 @@ def retrieve_jwt_expiration_timestamp(jwt_value):
         raise ValueError("Invalid JWT structure. Expected a JWS Compact Serialization formatted value.")
 
     try:
-        padded_base64_payload = base64.b64decode(parts[1]).decode('utf-8')
+        padded_base64_payload = base64.b64decode(parts[1] + "===").decode('utf-8')
```

Comment:
Could you add a comment what this does/ why it's needed? Point to https://tools.ietf.org/id/draft-jones-json-web-token-02.html#base64urlnotes  

I would be surprised/confused what this does

Reply:
Good call. Will do.